10-std.tex

```diff
@@ -295,7 +295,79 @@ \subsection{Extensions}
 \section{Lambda}
 \label{std-Lambda}
 
+\define{Lambda}{define-lambda}{Lambda is a functional language concept within Haxe that allows you to apply a function to a set or \tref{custom iterators}{lf-iterators}. The Lambda class is a collection of functional methods in order to use functional-style programming with Haxe.}
 
+It is ideally used with \expr{using Lambda} (read \tref{Static Extension}{lf-static-extension}) and then acts as an extension to \type{Iterable} types. 
+
+On static platforms, working with the \type{Iterable} structure might be slower than performing the operations directly on known types, such as \type{Array} and \type{List}.
+
+\paragraph{Lambda Functions}
+The Lambda class allows us to operate on an entire \type{Iterable} at once.
+This is often preferable to looping routines since it is less error prone and easier to read. 
+For convenience, the \type{Array} and \type{List} class contains some of the frequently used methods from the Lambda class.
```

Comment:
one newline means nothing, two means a new paragraph. so it is safe to use it. I found it a bit easier to read in my editor.